### PR TITLE
Only allow to open web app if .todos.json exists in current directory

### PR DESCRIPTION
**Pull request by Bunchhieng**

### CODE REVIEW

File: todo.go [hunk 1]
```diff
@@ -128,9 +128,22 @@ func routeInput(command string, input string) {
...
-		open.Start("http://localhost:7890")
-		web.Run()
+		// Only open default browser if .todos.json exists
+		pwd, err := os.Getwd()
+		if err != nil {
+			fmt.Println(err)
+			os.Exit(1)
+		}
+
+		if _, err := os.Stat(pwd + "/.todos.json"); err == nil {
```
Comment (gammons) [inline todo.go]: You can get this functionality by instantiating a FileStore and running [Load](https://github.com/gammons/todolist/blob/master/todolist/file_store.go#L19) on it.  That will cause the program to exit if there is not a .todos.json in the cwd.  Then you're safe to start the webapp after that.

**Comment by Bunchhieng** [comment]
Sorry for late response. Looking on it @gammons

**Comment by Bunchhieng** [comment]
working on fixing test failing.

**Comment by gammons** [comment]
Got a little bit bigger!  I think it looks ok.  going to test drive and double check later tonight.

**Comment by gammons** [comment]
Sorry for the late merge.  Looks good!

**Pull request merged**